### ISSUE TITLE
a mistyped error

### ISSUE THREAD

**Issue by yifungkhong** [NanoMichael/MicroTeX#148]
https://github.com/NanoMichael/MicroTeX/blob/e3199b8b80d58717440a506fc7523d9f01b49bd6/src/atom/atom_char.cpp#L34

it is not belonged to symbolFormulaMappings, the statement is always true?

**Comment by NanoMichael** [comment]
You are right. This code is searching in the cache to see if a 'SymbolAtom' has already been created, which does not affect the main process 😄.

By the way, the `master` branch is going to be deprecated, if you're interested, take a look at `openmath` branch.
